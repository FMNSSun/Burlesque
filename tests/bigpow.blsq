{ri1R@{2\/**}m[\/{.>}\/+]fi1R@{2\/**}m[\/!!Sh}WL