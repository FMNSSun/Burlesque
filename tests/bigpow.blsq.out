4
2
8
2
8
16
4
8
8
4